src/link/MachO/Dylib.zig

```diff
@@ -71,9 +73,48 @@ pub fn closeFile(self: Dylib) void {
     }
 }
 
+fn decodeArch(cputype: macho.cpu_type_t) !std.Target.Cpu.Arch {
+    const arch: std.Target.Cpu.Arch = switch (cputype) {
+        macho.CPU_TYPE_ARM64 => .aarch64,
+        macho.CPU_TYPE_X86_64 => .x86_64,
+        else => |value| {
```

Comment:
Could you move the `log.err` informing which arch was passed to here?

Reply:
The CI is failing since this is an used capture here. You capture any other potential architecture as `value` but never use it. In this case, just omit the capture altogether like so:

```zig
else => {
```

Reply:
The CI is failing since this is an used capture here. You capture any other potential architecture as `value` but never use it. In this case, just omit the capture altogether like so:

```zig
else => {
```